packages/theming/atlas/src/theme/native/custom-variables.ts

```diff
@@ -500,7 +500,8 @@ export const floatingActionButton: VariablesFloatingActionButton = {
         fontSize: font.sizeSmall
     },
     secondaryButtonCaptionContainer: {
-        backgroundColor: background.primary
+        backgroundColor: background.primary,
+        borderColor: darkMode ? background.primary : "#eee"
```

Comment:
I think it makes more sense to just use `background.primary` for the border as well, regardless of the theme. So if someone changes `background.primary` they won't run into the same problem because the styles will automatically adapt.

Reply:
Or alternatively, use border.color if it looks good

Reply:
I changed to use `background.primary` 👍